<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureConverter</class>
 <widget class="QMainWindow" name="TemperatureConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperature Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_input">
       <property name="text">
        <string>Input Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_input">
       <property name="toolTip">
        <string>Enter a number to be converted</string>
       </property>
       <property name="inputMask">
        <string extracomment="Input must be a number"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_convert">
       <property name="text">
        <string>CONVERT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_result">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_result">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="gr_input_type">
       <property name="title">
        <string>Input scale</string>
       </property>
       <widget class="QRadioButton" name="rb_in_fahrenheit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Fahrenheit</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_in_celsius">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Celsius</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_in_kelvin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Kelvin</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gr_output_type">
       <property name="title">
        <string>Output scale</string>
       </property>
       <widget class="QRadioButton" name="rb_out_fahrenheit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Fahrenheit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_out_celsius">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Celsius</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_out_kelvin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Kelvin</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>TemperatureConverter</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
